common/src/main/java/com/wynntils/overlays/StatusEffectsOverlay.java

```diff
@@ -207,5 +249,44 @@ public void setCount(int c) {
         public StatusEffect getEffect() {
             return this.effect;
         }
+
+        public void addModifier(StyledText modifier) {
+            this.modifierList.add(modifier.getStringWithoutFormatting());
+        }
+
+        private double extractDoubleFromString(String string) {
+            byte[] s = string.getBytes();
+            int len = string.length();
+
+            int start = 0;
+            while (!Character.isDigit(s[start]) && s[start] != '-') {
+                start += 1;
+            }
+
+            int end = start;
+            while (end < len && (Character.isDigit(s[end]) || s[end] == '.')) {
+                end += 1;
+            }
+            return Double.parseDouble(string.substring(start, end));
+        }
+
+        private int indexAfterDigits(String string) {
+            byte[] s = string.getBytes();
+            int len = string.length();
+            int i = 0;
+            while (!Character.isDigit(s[i])) {
+                i += 1;
+            }
+            while ( i < len && (Character.isDigit(s[i]) || s[i] == '.')) {
+                i += 1;
+            }
+            return i;
+        }
+    }
+
+    private enum StackingBehaviour {
+        NONE,
+        MULTIPLIER,
```

Comment:
Using "multiplier" next to "sum" sounds like you are multiplying effects rather than adding them, which would be just weird. 

What do you think about renaming MULTIPLIER to GROUP?

Reply:
I agree that `GROUP` is more descriptive than `MULTIPLIER`, changing it